flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/codegen/CalcCodeGenerator.scala

```diff
@@ -69,6 +71,42 @@ object CalcCodeGenerator {
     new OneInputOperatorWrapper(genOperator)
   }
 
+  private[flink] def generateFunction[T <: Function](
```

Comment:
why don't adding this to `FunctionCodeGenerator`?

Reply:
Because it actually generate code for a `CalcProgram`, it needs to access the private method `generateProcessCode` in this file. And `FunctionCodeGenerator` only accepts code body as  parameter, not the `CalcProgram.

What do you think about renaming the method name to `generateCalcFunction` to align with `generateCalcOperator` ?